pkg/yurthub/filter/initializer/initializer.go

```diff
@@ -45,9 +45,9 @@ type WantsNodePoolName interface {
 	SetNodePoolName(nodePoolName string) error
 }
 
-// WantsStorageWrapper is an interface for setting StorageWrapper
-type WantsStorageWrapper interface {
-	SetStorageWrapper(s cachemanager.StorageWrapper) error
```

Comment:
It may not be a good idea to change the interface, since most of its capability is desigened for handling and caching request. The original handle flow is:
```
 req -> yurthub proxy server -> filters -> cachemanager
```

if filters also consist of cachemanager, it will be:
```
 req -> yurthub proxy server -> filters -> cachemanager -> cachemanager
```

There may be unexpected problems in yurthub cache.

Reply:
This interface is only use in filter which want to get kubelet node data from disk stroge, and we discuss to use cachemanager in filter, there is no way to find the cachemanager in filter now, and if we use cachemanger, the storage is out of use.

What do you think, save this interface and create a use one?